src/providers/completion/styleCompletionItemProvider.ts

```diff
@@ -194,8 +194,9 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
 	 *
 	 * @returns {Array}
 	 */
-	public getPropertyValueCompletions (linePrefix, prefix) {
+	public getPropertyValueCompletions (linePrefix, prefix, position, document) {
 		const { properties } = this.completions.titanium;
+		const range = document.getWordRangeAtPosition(position, /([\w\"\.\'\$]+)/);
```

Comment:
I think you can skip the escaping when using a special character in character sets?